src/pages/dho/Configuration.vue

```diff
@@ -102,18 +102,18 @@ export default {
 
   data () {
     return {
-      tab: 'GENERAL',
-      tabs: ['GENERAL', 'VOTING', 'COMMUNITY', 'COMMUNICATION', 'DESIGN', 'PLAN'],
+      confirmLeavePage: null,
 
       form: {},
       initialForm: {},
 
-      confirmLeavePage: null
+      tab: 'GENERAL',
+      tabs: ['GENERAL', 'VOTING', 'COMMUNITY', 'COMMUNICATION', 'DESIGN', 'PLAN']
```

Comment:
@arsenijesavic In general all those should be constants, since you are using them later in the code